### PR TITLE
NEW: Add blog module

### PR DESCRIPTION
**Pull request by unclecheese**
Resolves https://github.com/silverstripeltd/open-sourcerers/issues/76

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,13 +18,15 @@
         "silverstripe/crontask": "^2.1.2",
         "silverstripe/dynamodb": "^4.0.1",
         "dnadesign/silverstripe-elemental-userforms": "^3.0.0",
-        "jonom/silverstripe-betternavigator": "^4.1",
+        "jonom/silverstripe-betternavigator": "dev-pulls/auto-injection",
```
Comment (maxime-rainville) [inline composer.json:21]: Does this change relate to blogs or is it just cleaning up other stuff?

**Review by maxime-rainville — APPROVED** [review]

**Comment by maxime-rainville** [comment]
Travis is grumpy about fetching the PR branch.

```
[RuntimeException]                                                           
  Failed to execute git clone --mirror 'git@github.com:/unclecheese/silverstr  
  ipe-betternavigator' '/home/travis/.composer/cache/vcs/git-github.com--uncl  
  echeese-silverstripe-betternavigator/'         
```

File: composer.json (hunk 1 above)
Comment (unclecheese) [reply composer.json:21]: Yeah, believe it or not, it does. It was causing the sidebar not to show, for reasons that aren't worth getting into.

**Comment by maxime-rainville** [comment]
Rebased and updated to use the latest version of theme.

Looks like the build is still stuffed.

**Pull request merged**